documentation/source/install/install_overview.rst

```diff
@@ -1,3 +1,8 @@
-Setting Up OpenEBS Overview
-==========================
+Setting Up OpenEBS -  Overview
+==============================
 
+OpenEBS can run on various platforms: from your laptop, to VMs on a cloud provider. OpenEBS can be setup as a single node or multi-node deployments.
+
+OpenEBS runs on Google and Amazon Cloud providers and is tested on Ubuntu and CoreOS.
+ 
+Users with existing data center infrastructure may prefer using on-premise or hybrid (data distributed between on-premise and cloud environments) solutions.
```

Comment:
The content can be re-structured as follows:

OpenEBS can run on various platforms: from your laptop, to VMs on a cloud provider. OpenEBS is also aimed at providing the option of using hybrid deployments where data distributed between on-premise and cloud environments.

If you are beginner to Kubernetes and OpenEBS, we recommend you get started with either running an OpenEBS cluster on your laptop using Vagrant or if you have access to the Cloud, you can use our custom solutions for Google and Amazon Cloud providers.  The kubernetes setup guide (link to the kuberentes setup), provides you with many more options to choose from for installing kubernetes. 

If you are already an experienced Kubernetes user, you can easily setup OpenEBS on your existing kubernetes cluster with a few simple kubectl commands. (link to the instructions in the quick started guide or replicate the instructions here). 

We are looking for help from the community in including additional platforms where OpenEBS has been successfully deployed. Please share your story through Github Issue(github issues link) or PR .

Reply:
OpenEBS Issues link : https://github.com/openebs/openebs/issues

Kubernetes Setup Link : https://kubernetes.io/docs/setup/